lib/backend/report.go

```diff
@@ -676,7 +676,7 @@ func (r *ReporterWatcher) watch(ctx context.Context) {
 }
 
 var (
-	requests = prometheus.NewCounterVec(
+	Requests = prometheus.NewCounterVec(
```

Comment:
All these exported names from the lib/backend package (which is widely imported) make me a bit uneasy, especially because they have very generic-sounding names like "Writes". Can we move them to a `backendmetrics` package or something along those lines?